specifyweb/frontend/js_src/lib/components/FormSliders/IntegratedRecordSelector.tsx

```diff
@@ -252,7 +256,8 @@ export function IntegratedRecordSelector({
                 preHeaderButtons={collapsibleButton}
                 sortField={sortField}
                 viewName={viewName}
-                onAdd={(resources): void => {
+                onAdd={isTaxonTreeDefItemTable ? undefined :
```

Comment:
considering that the TaxonTreeDefItems have their own specific adding setup, we did this so that there wouldn't be any weirdness with ranks (if you add a rank via the TreeDef editor then the new rank will be the lowest rank + 10 automatically and you can't edit it unless forms have been changed)

We don't think our changes will affect anything else but let us know 👍